client/my-sites/patterns/pages/category/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useLocale } from '@automattic/i18n-utils';
+import {
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+import { CategoryNavigation } from 'calypso/components/category-navigation';
+import DocumentHead from 'calypso/components/data/document-head';
+import { PatternsGetStarted } from 'calypso/my-sites/patterns/components/get-started';
+import { PatternsHeader } from 'calypso/my-sites/patterns/components/header';
+import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/controller';
+import { usePatternCategories } from 'calypso/my-sites/patterns/hooks/use-pattern-categories';
+import { usePatterns } from 'calypso/my-sites/patterns/hooks/use-patterns';
+import ImgGrid from './images/grid.svg';
+import ImgStar from './images/star.svg';
+import type { PatternGalleryFC } from 'calypso/my-sites/patterns/types';
+
+import './style.scss';
+
+type PatternsCategoryPageProps = {
+	category: string;
+	isGridView?: boolean;
+	patternGallery: PatternGalleryFC;
+};
+
+export const PatternsCategoryPage = ( {
+	category,
+	isGridView,
+	patternGallery: PatternGallery,
+}: PatternsCategoryPageProps ) => {
+	const locale = useLocale();
+
+	const { data: categories } = usePatternCategories( locale, RENDERER_SITE_ID );
+	const { data: patterns } = usePatterns( locale, category );
+
+	const categoryNavList = categories?.map( ( category ) => ( {
+		name: category.name || '',
+		label: category.label,
+		link: `/patterns/${ category.name }`,
+	} ) );
+
+	return (
+		<>
+			<DocumentHead title="WordPress Patterns- Category" />
+
+			<PatternsHeader
+				title={ category + ' page patterns' }
```

Comment:
```suggestion
				title={ category + ' patterns' }
```
Minor, and I realize that this isn't final regardless, but let's skip `page` here.

Reply:
Something to keep in mind for the future: we won't be able to concatenate content like this when we localize that title. We can certainly use a placeholder but if we want to optimize that page for search engines we will probably have to follow the approach used in the `Themes` page where a different title is defined [statically](https://github.com/Automattic/wp-calypso/blob/975b3c2a995d3f39bd8e1904d1b90c0276133116/client/my-sites/themes/use-theme-showcase-logged-out-seo-content.js#L26) for each category.

Reply:
Thanks @stephanethomas for point to nice example.
The current approach is just a dummy copy, to render different headers.

Reply:
I was thinking about how we should handle translations. Thanks for digging up that reference, @stephanethomas 👍